### DISCUSSION TITLE
VScode Extensions on Zed

### DISCUSSION THREAD

**Discussion by qvipin** [zed-industries/zed#17387] [Feature Requests]
### Check for existing issues

- [x] Completed

### Describe the feature

I want to switch to Zed because it looks really really good, however there are certain extensions I wish were in Zed but aren't so I cannot switch. Is there any progress towards implementing this?

### If applicable, add mockups / screenshots to help present your vision of the feature

N/A

**Comment by SomeoneToIgnore** [comment]
> certain extensions

Can you elaborate?
At this level of detail, I can only say that some progress had been made.

**Reply by qvipin** [reply]
> > certain extensions
> 
> Can you elaborate? At this level of detail, I can only say that some progress had been made.

there are certain extensions like mdx support, specific pico development extensions, github extensions like github repos if i dont want to clone the repo, remote repos, remote ssh (makes life easier), and im pretty sure stuff like python debugging exist but dont seem to find it

**Comment by qvipin** [comment]
There are  just certain items that are already so well made in vscode and maybe it would be nice if we had a way of porting them

**Reply by SomeoneToIgnore** [reply]
I see, so this is https://github.com/zed-industries/zed/issues/4845 then, and there's been no progress on that front.
I doubt that it will be possible anytime soon, as Zed's extension API is very limited currently, and other parts are not stabilized yet.
In particular (but not limited to), currently there's 0 support for task/command registration; editor or any graphical manipulations.
